contracts/HubPool.sol

```diff
@@ -4,9 +4,13 @@ pragma solidity ^0.8.0;
 import "@uma/core/contracts/common/implementation/Testable.sol";
 import "@uma/core/contracts/common/implementation/Lockable.sol";
 import "@uma/core/contracts/common/implementation/MultiCaller.sol";
+import "@uma/core/contracts/common/implementation/ExpandedERC20.sol";
```

Comment:
nit: something we might consider for a follow-up. We could swap this out for an ERC20 contract that's optimized for this use case (immutable and singular minter/burner).